tests/integration/test_tests.rs

```diff
@@ -1,12 +1,12 @@
 // Copyright 2018-2025 the Deno authors. MIT license.
 
-use test_util as util;
```

Comment:
I don't like this util alias because auto-imports for Rust never uses it so it causes a mix of util and test_util. Might as well just always use test_util.